src/core/Akka.API.Tests/CoreAPISpec.ApproveClusterTools.approved.txt

```diff
@@ -802,22 +165,22 @@ namespace Akka.Cluster.Tools.PublishSubscribe
 {
     public sealed class CurrentTopics : System.IEquatable<Akka.Cluster.Tools.PublishSubscribe.CurrentTopics>
     {
-        public readonly string[] Topics;
         public CurrentTopics(string[] topics) { }
+        public string[] Topics { get; }
```

Comment:
I'm not fan of exposing arrays in public API. Could I change it to IReadOnlyCollection<string> ?